run_squad.py

```diff
@@ -154,10 +159,13 @@
     "null_score_diff_threshold", 0.0,
     "If null_score - best_non_null is greater than the threshold predict null.")
 
-flags.DEFINE_string("optimizer_type", "adam", "Optimizer used for training - adam (default), lamb, nadam and nlamb")
+flags.DEFINE_string(
+    "optimizer_type", "adam", "Optimizer used for training - adam (default), lamb, nadam and nlamb")
```

Comment:
unnecessary change

Reply:
Thanks a lot, Michael. That's probably from the autoformatter because the line is too long.

Maybe we want to keep the line that way for now. I don't know how to pick and choose within a commit.